modules/ROOT/pages/comments-commands-events-apis.adoc

```diff
@@ -10,3 +10,30 @@
 The Comments plugin provides the following {productname} commands.
 
 include::partial$commands/comments-cmds.adoc[]
+
+== API
+
+The Comments plugin exposes the API `editor.plugins.tinycomments.getEventLog()` which returns a log of all `CommentChange` events. These events are dispatched whenever:
+
+* A new comment is added (and who added it).
+* A comment is edited (and who edited it).
+* A reply to a comment is added (and who added the reply).
+* A comment is resolved (and who resolved the comment).
+* A comment is deleted (and who deleted the comment).
+
+:minimumversion: 6.1
+include::partial$misc/minimum-version.adoc[]
+
+The event log can be retrieved either in full or with the `after` option, which restricts the returned list to Comment events after a time-stamp date in the link:https://www.iso.org/standard/70907.html[ISO-8601] format, both shown in the following:
+
+=== Example: Using the `+comments+` plugin APIs
+
+[source,js]
+----
+const comments = tinymce.activeEditor.plugins.tinycomments;
+
+console.log(comments.getEventLog());
+console.log(comments.getEventLog(
+    { after: '2022-02-22' }     //ISO-8601 standard: YYYY-MM-DD
```

Comment:
I suggest including some minutes, at least, since it's unlikely customers will want to use date granularity for the API calls